src/components/templates/Auditorium/components/Section/Section.tsx

```diff
@@ -58,14 +71,27 @@ export const Section: React.FC<SectionProps> = ({ venue }) => {
   const iframeInlineStyles: React.CSSProperties = useMemo(
     () => ({
       width: `calc(${videoWidthInSeats} * (${SECTION_SEAT_SIZE} + ${SECTION_SEAT_SPACING}))`,
-      height: `calc(${videoHeightInSeats} * (${SECTION_SEAT_SIZE} + ${SECTION_SEAT_SPACING}))`,
+      height: `calc(${
+        videoHeightInSeats - REACTIONS_CONTAINER_HEIGHT_IN_SEATS
+      } * (${SECTION_SEAT_SIZE} + ${SECTION_SEAT_SPACING}))`,
       minWidth: `calc(${videoWidthInSeats} * (${SECTION_SEAT_SIZE_MIN} + ${SECTION_SEAT_SPACING}))`,
-      minHeight: `calc(${videoHeightInSeats} * (${SECTION_SEAT_SIZE_MIN} + ${SECTION_SEAT_SPACING}))`,
+      minHeight: `calc(${
+        videoHeightInSeats - REACTIONS_CONTAINER_HEIGHT_IN_SEATS
+      } * (${SECTION_SEAT_SIZE_MIN} + ${SECTION_SEAT_SPACING}))`,
     }),
     [videoWidthInSeats, videoHeightInSeats]
   );
 
+  const reactionsContainerInlineStyles: React.CSSProperties = useMemo(
+    () => ({
+      width: `calc(0.9 * ${videoWidthInSeats} * (${SECTION_SEAT_SIZE} + ${SECTION_SEAT_SPACING}))`,
+      minWidth: `calc(0.9 * ${videoWidthInSeats} * (${SECTION_SEAT_SIZE_MIN} + ${SECTION_SEAT_SPACING}))`,
+    }),
+    [videoWidthInSeats]
+  );
```

Comment:
I'm not sure it's the best way to display it. There should be a more neat way

Reply:
Not using inline styles ideally. Probably can use similar patterns to what we use in `Audience` with `useCss` / etc if we need to pass things between JS / SCSS.

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1339/commits/66819ea73f97abf7d613bcc012802229098bee29

Reply:
Done in https://github.com/sparkletown/sparkle/pull/1339/commits/d2a708581428aa087efe8271fe99a7fa1029bdbe